src/DustExtinction.jl

```diff
@@ -252,17 +253,17 @@ function ebv_galactic(dustmap::SFD98Map, l::Real, b::Real)
         val = (1. - xw) * data[1] + xw * data[2]
     else
         data = read(hdu, x0:x0+1, y0:y0+1)
-        val = ((1.-xw) * (1.-yw) * data[1, 1] +
-               xw      * (1.-yw) * data[2, 1] + 
-               (1.-xw) * yw      * data[1, 2] +
+        val = ((1. -xw) * (1. -yw) * data[1, 1] +
```

Comment:
I'd not use `1.`, better an explicit `1.0` (or probably `1` is even _more_ better)